_posts/abaker8/2017-05-27-abaker8-amandas-custome-turtle.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+author: abaker8
+title: "Amandas custome turtle"
+---
+Here is my custome turtle!!
+
+<iframe src="https://trinket.io/embed/python/3af593fea4" width="100%" height="600" frameborder="0"
+
+
+This custome turtle was tuns of fun and  the only frustrating part was lining up the angles for the triangles. They may not be perfect, but that was geometry I thought i would never have to do again!!! However, I really liked coding the different functions into the turtle so that way I wasnt just offering the person to choose the color of the turtle but it used logic so they had alternatives. One thing I am not sure a "sighted person  can see, and feedback would be great, is the few times I had tina use the write function. So written inside the blue triangle in black is the word "blue" and the red triangle has the word "red" written in it. Depending on what the user chooses when they pick a color, if they choose blue, she will write at (0, 340) Tina likes the color blue in lavender ink. If the user chooses red, she fills the screen with black and writes at (0, 0) in red ink, "Tina doesn't like the color red very much!!" Let me know if that is evident for someone looking at the screen. I am excited to do future turtle exercises with other cool effects! 
```

Comment:
Very creative custom turtle!

You did a great job using functions well here.  Your 'hidden' messages are very creative as well.  

There are two things that probably weren't apparent to you.  These items are just FYIs
First off, turtletalk had a bug with writing - it wasn't doing it, as you found out in a later exercise.  Sorry!  This made what it was telling you false (though when the fix gets applied all will be well)

It strikes me that with lining up your triangles, `goto` may be much easier than headings and lengths.

You've done a great job having fun with the assignment, using the tools we had available for this assignment, and putting your personality into it. Nice work.